packages/authentication/src/metadata.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as assert from 'assert';
+import 'reflect-metadata';
+
+export function authenticate(controllerClass: any, methodName: string) {
```

Comment:
I think it will be better to make `authenticate` a factory so that we can do things such as:
```
@authenticate('my-strategy', my-options)
```